src/patterns/validation/index.md.njk

```diff
@@ -39,6 +39,12 @@ Validation should refuse to accept:
 
 For example, if you’re asking for someone’s date of birth you should not accept ‘13’ in the month field.
 
+Remove or ignore any whitespace in users’ answers where it’s not needed. If an invisible character causes a validation error, it can be hard for users to know what's gone wrong and how to fix it. Consider removing:
```

Comment:
It might be helpful to start this sentence with 'Validation should accept:', to differentiate it from the earlier part of this section that talks about things you shouldn't accept.

We seem to use the wording "accept postcodes with or without spaces" at the start of the page.

Reply:
Consider replacing 'where it's not needed' with 'unless it makes the information too ambiguous for you to use'.